packages/dev/core/src/Audio/audioSceneComponent.ts

```diff
@@ -222,6 +228,33 @@ Object.defineProperty(Scene.prototype, "audioListenerPositionProvider", {
     configurable: true,
 });
 
+Object.defineProperty(Scene.prototype, "audioListenerRotationProvider", {
+    get: function (this: Scene) {
+        let compo = this._getComponent(SceneComponentConstants.NAME_AUDIO) as AudioSceneComponent;
+        if (!compo) {
+            compo = new AudioSceneComponent(this);
+            this._addComponent(compo);
+        }
+
+        return compo.audioListenerRotationProvider;
+    },
+    set: function (this: Scene, value: () => Vector3) {
+        let compo = this._getComponent(SceneComponentConstants.NAME_AUDIO) as AudioSceneComponent;
+        if (!compo) {
+            compo = new AudioSceneComponent(this);
+            this._addComponent(compo);
+        }
+
+        if (typeof value !== "function") {

```

Comment:
Why an explicit test when `value` is declared as `() => Vector3`?

Reply:
no idea :-) just did the exact same than the position one for consistency :-)